docs/2.9.0/docs/query/pqs-user-guide.rst

```diff
@@ -505,988 +1780,2081 @@ Use a ``--config`` file to define multiple options or reflect an infrastructure-
-
-The ``-pipeline-ledger-stop`` argument is an enum with the following possible values:
-
--  ``Latest``: Stop reading stream when latest offset is reached.
--  ``Never``: Continue reading stream indefinitely.
-
---pipeline-datasource
----------------------
-
-The ``-pipeline-datasource`` argument is an enum with the following possible values:
-
--  ``TransactionStream``: Read the ledger's filtered transaction stream for a set of parties. Includes creates, archives, and interface views, but excludes transient contracts and exercises. Transient contracts are contracts that were both created and archived in the same transaction.
--  ``TransactionTreeStream``: Read the ledger's transaction tree stream for a set of parties. Includes creates, exercises, and transient contracts, but excludes interface views.
-
---pipeline-filter
------------------
-
-The ``--pipeline-filter string`` option needs a filter expression to determine
-which templates and interfaces to include. A filter expression is a simple wildcard
-inclusion statement with basic Boolean logic, where whitespace is ignored. Below are some examples:
-
-- ``*``: everything, which is the default
-- ``a.b.c.Bar``: just this one fully qualified name
-- ``a.b.c.*``: all under the ``a.b.c`` namespace
-- ``deadbeef..:a.b.c.Foo`` just this one fully qualified name from this specific package ID
-- ``!a.b.c.Bar``: everything except this fully qualified name
-- ``a.b.c.Foo & a.b.c.Bar``: this is an error because it can't be both
-- ``(a.b.c.Foo | a.b.c.Bar)``: these two fully qualified names
-- ``(a.b.c.* & !(a.b.c.Foo | a.b.c.Bar) | g.e.f.Baz)``: everything in ``a.b.c`` except for ``Foo`` and ``Bar``, and also include ``g.e.f.Baz``
-
---pipeline-parties
-------------------
-
-The ``--pipeline-parties`` option supports the same filter expressions as the
-``--pipeline-filter``. So to filter for two parties ``alice::abc123...`` and
-``bob::def567...``, you could write ``--pipeline-parties="(alice* | bob*)"``.
-If you want to specify a specific party, include the hash behind the party
-hint (such as
-``Alice_1::122055fc4b190e3ff438587b699495a4b6388e911e2305f7e013af160f49a76080ab``).
-
-Please note that the separator is a pipe character (``|``) instead of a comma.
-
-Brackets are unnecessary for simple expressions. A simple list is
-``--pipeline-parties="Alice_1::122055fc4b190e3ff438587b699495a4b6388e911e2305f7e013af160f49a76080ab
-|
-Alice_2::122053933e4803c2995e41faa8a29981ca0d1faf6b4ffbf917ba1edd0db133acb634
-| Peter-1::358400000000000000000000000`` Specifying the parties in a short
-form can be done by using the ``*`` as a wildcard. For example,
-``--pipeline-parties="Alice* | *358400000000000000000000000"`` selects
-``Alice_1``, ``Alice_2``, and ``Peter-1``.
+Commonly used configuration arguments include:
 
 More advanced expressions can make use of brackets, such as
 ``--pipeline-parties="Alice* | Bob* | (participant* & !(participant3::*))"``.
 
-
 Handle Configuration Changes
```

Comment:
This really needs to be much earlier.  It is very important information.  I'd bet a months wages that the reader who is trying to install PQS won't get to this point.

Reply:
I'm not sure where it can go - it is important, but you need to understand it in the context of what configuration is?